src/components/publishing/sections/video.tsx

```diff
@@ -132,7 +129,7 @@ const PlayButton = styled.div`
 `
 
 const sizeMeOptions = {
-  refreshRate: 64,
+  refreshRate: 500,
```

Comment:
Should this be a global constant?

Reply:
Yeah, maybe it's time we add that constants file - will add now!